changelogs/fragments/T7260-remove-last-firewall-group-member.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```

Comment:
If this is a fix, then this should be a bug fix not a trivial change.

Reply:
Oops, copy/pasted without reading :facepalm: It's fixed now.

Reply:
In thr future, please just add a commit to your branch and commit.  That way we see the updates in place.  We always squash the commits on merge anyway.